src/content/docs/query-your-data/nrql-new-relic-query-language/get-started/introduction-nrql-tutorial.mdx

```diff
@@ -0,0 +1,480 @@
+
+import queriesnrqltutorial18 from 'images/queries-nrql_screenshot-crop-tutorial-18.png'
+
+import queriesnrqltutorial19 from 'images/queries-nrql_screenshot-crop-tutorial-19.png'
+
+import queriesnrqltutorial20 from 'images/queries-nrql_screenshot-crop-tutorial-20.png'
+
+We've covered the basics of NRQL in our other topics, such as [what NRQL is](/docs/query-your-data/nrql-new-relic-query-language/get-started/introduction-nrql-new-relics-query-language/), [how NRQL works](docs/query-your-data/nrql-new-relic-query-language/get-started/introduction-nrql-how-nrql-works/), and [how NRQL relates to charts and dashboards](docs/query-your-data/nrql-new-relic-query-language/get-started/charts-and-dashboards-with-nrql/). With this information, beginning to work with NRQL is the next step, and this tutorial is designed to help you do just that! Below we'll cover some foundational concepts within the language and get you set up with creating a some basic NRQL queries using your data. This tutorial teaches you how to:
+
+* Use `SELECT` and `FROM`
+* Use `LIMIT`
+* Specify attributes
+* Aggregate using `average()`, `max()`, `min()`, `sum()`, and `count()`
+* Use `SINCE` and `UNTIL`
+* Plot line graphs with `TIMESERIES`
+* Filter using `WHERE`
+* Group attributes using `FACET`
+
+Want to learn NRQL within the New Relic platform? We've included a NRQL Lessons application on our [opensource.newrelic.com](https://opensource.newrelic.com/projects/newrelic/nr1-learn-nrql) site to help you quickly see NRQL's value and power within the application. The video below shows you how to install the NRQL Lesson.
+
+<Video
+  id="3UHEHbprs6I"
+  type="youtube"
+/>
+
+Ready to get started? Let's go!
+
+<CollapserGroup>
+  <Collapser
+    className="freq-link"
+    id="first-query"
+    title="1. Your First Query"
+  >
+  <SideBySide>
+  <Side>
+Let's start by looking at a single event type in NRDB called **Transaction**, gathered by New Relic APM. Every NRQL query must have `SELECT` and `FROM` clauses: you must `SELECT` some data and tell us where it's `FROM`. Start with this basic query that selects everything from the **Transaction** event type.
+ 
+    ```sql
+    SELECT *
+    FROM Transaction
+    ```
+
+  </Side>
+  <Side>
+    <img
+  title="Select and From query"
+  alt="A screenshot of a query with Select and From"
+  src={queriesnrqltutorial1}
+/>
+  </Side>
+  </SideBySide>
+  <SideBySide>
+  <Side>
+ This returns a lot of results, each with a timestamp and a collection of attributes. For now, we only want a single result, so let's limit the results to a single record using `LIMIT 1`.
+
+ <Callout variant="important"> When a LIMIT isn't supplied the default will be used, which is 100 table rows for SELECT * queries or 10 aggregated values for FACET queries and SELECT (attributes) queries. You can specify any limit up to the maximum. Use LIMIT MAX to return the maximum number of results possible.
```

Comment:
A bit stumped on this callout, I feel like it looks kind of weird here (it looks worse after the query where I originally had it), but it seems important enough to know this information that some sort of callout should be made. Any suggestions are more than welcome.

Reply:
might consider using some code formatting for some of this, like 

`SELECT *` queries
`SELECT (attributes)` queries

Reply:
Thank you!